projects/ng-components/README.md

```diff
@@ -1,25 +1,82 @@
 # SEB Angular Components
 
-This library was generated with [Angular CLI](https://github.com/angular/angular-cli) version 10.0.2.
+![Deployment](https://github.com/sebgroup/ng-components/workflows/Deployment/badge.svg)
+![npm](https://img.shields.io/npm/v/@sebgroup/ng-components?color=brightgreen)
+![Github Pages](https://github.com/sebgroup/ng-components/workflows/Github%20Pages/badge.svg)
+[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+[![Coverage Status](https://coveralls.io/repos/github/sebgroup/ng-components/badge.svg?branch=master)](https://coveralls.io/github/sebgroup/ng-components?branch=master)
+[![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=sebgroup/ng-components)](https://dependabot.com)
 
-## Code scaffolding
+This is a set of angular components some of which are based on SEB's bootstrap. The plan for this project is to increase and improve components for future usage.
 
-Run `ng generate component component-name --project ng-components` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module --project ng-components`.
+-   The package name: `@sebgroup/ng-components`
+-   The package documentation: [Documentation](https://sebgroup.github.io/ng-components)
+-   The package sourcecode: [Github Source Code](https://github.com/sebgroup/ng-components)
+-   NPM package: [@sebgroup/ng-components](https://www.npmjs.com/package/@sebgroup/ng-components)
```

Comment:
Can you add links to [Demo](https://sebgroup.github.io/ng-components/) and [Getting Started](https://sebgroup.github.io/ng-components/docs/getting-started) pages